src/search.hpp

```diff
@@ -51,11 +51,15 @@ template <DbIndexType   c_indexType,
           AlphabetEnum  c_origSbjAlph>
 void argConv2(LambdaOptions const & options);
 
+template <DbIndexType   c_indexType,
+          AlphabetEnum  c_origSbjAlph>
+void argConv3(LambdaOptions const & options);
```

Comment:
Here, as well. If they are not called in succession, I would not number them increasingly. The whole thing is confusing enough as it is :wink: